packages/ecma402-abstract/NumberFormat/InitializeNumberFormat.ts

```diff
@@ -114,34 +121,100 @@ export function InitializeNumberFormat(
     notation
   )
 
-  const compactDisplay = GetOption(
+  let roundingIncrement = GetNumberOption(
+    options,
+    'roundingIncrement',
+    1,
+    5000,
+    1
+  )
+
+  if (VALID_ROUND_INCREMENT_VALUES.indexOf(roundingIncrement) === -1) {
+    throw new RangeError(
+      `Invalid rounding increment value: ${roundingIncrement}.
+      Valid values are ${VALID_ROUND_INCREMENT_VALUES}.`
+    )
+  }
+
+  if (
+    roundingIncrement !== 1 &&
+    internalSlots.roundingType !== 'fractionDigits'
+  ) {
+    throw new TypeError(
+      `For roundingIncrement > 1 only fractionDigits is a valid roundingType`
+    )
+  }
+
+  if (
+    roundingIncrement !== 1 &&
+    internalSlots.maximumFractionDigits !== internalSlots.minimumFractionDigits
+  ) {
+    throw new RangeError(
+      'With roundingIncrement = 1, maximumFractionDigits and minimumFractionDigits must be equal'
+    )
+  }
+
+  internalSlots.roundingIncrement = roundingIncrement
+
+  let trailingZeroDisplay = GetOption(
+    options,
+    'trailingZeroDisplay',
+    'string',
+    ['auto', 'stripIfInteger'],
+    'auto'
+  )
+
+  internalSlots.trailingZeroDisplay = trailingZeroDisplay
+
+  let compactDisplay = GetOption(
```

Comment:
here also prefer const over let if the value is never mutated

Reply:
You're right, as the specification always says "let ..." sometimes I forget to convert them to const 😄